code/modules/organs/tendons.dm

```diff
@@ -0,0 +1,55 @@
+/datum/tendon
+	var/name = "tendon"
+	var/threshold
+	var/intact = TRUE
+
+	var/list/messages
+
+	var/obj/item/organ/external/parent
+
+/datum/tendon/New(var/obj/item/organ/external/E, var/name, var/damage_threshold, var/list/damage_msgs)
+	if(!istype(E))
+		crash_with("Tendon created with invalid parent organ: [E]")
+		qdel(src)
+		return
+
+	parent = E
+
+	if(name)
+		src.name = name
+	if(islist(damage_msgs))
+		messages = damage_msgs
+
+	if(damage_threshold)
+		threshold = damage_threshold
+	else
+		threshold = parent.min_broken_damage
+
+/datum/tendon/proc/heal()
+	if(!parent?.owner || intact)
+		return FALSE
+
+	. = intact = TRUE
```

Comment:
```suggestion
	intact = TRUE
	return TRUE
```

please 1 statement per line :<

Reply:
i have already altered the coding standards. pray i do not alter them further.

~~fixed.~~